ambry-frontend/src/test/java/com/github/ambry/frontend/NamedBlobPutHandlerTest.java

```diff
@@ -192,6 +193,26 @@ public void updateNamedBlobAllowedTest() throws Exception {
     assertEquals("Unexpected response status", restResponseChannel.getStatus(), ResponseStatus.Ok);
   }
 
+  @Test
+  public void testStripPrefixAndExtension() throws Exception {
```

Comment:
not exactly the test case that i am looking for. We have test cases in FrontendRestRequestServiceTest. Can we add a test of named blob put operation with cluster name and make sure this is correct?

Reply:
Added unit test as we discussed.